WATOLINK/bci-gaming
# Removed unused scenes, fixed menu position

## Not changing any functionality, just removing unused things from merges and fixing the way the menu is framed




## Steps to test
1. Make sure the menu is centered when you play and change screen size
2. Just make sure nothing else is broken


## Checklist
